src/server/integrations/handleGitHubWebhooks.js

```diff
@@ -0,0 +1,48 @@
+import crypto from 'crypto';
+import schema from 'server/graphql/rootSchema';
+import {graphql} from 'graphql';
+
+// TODO when this is all legit, we'll map through the queries & use the ASTs instead of the strings
+const eventLookup = {
```

Comment:
i don't love storing all this stuff in an object, but i think it's a lot cleaner than a conditional.